src/node_quic_session.cc

```diff
@@ -1014,12 +1014,20 @@ int QuicSession::OnRemoveConnectionID(
   return 0;
 }
 
-int QuicSession::OnPathValidation(
-    ngtcp2_conn* conn,
-    const ngtcp2_path* path,
-    ngtcp2_path_validation_result res,
-    void* user_data) {
-  // TODO(@jasnell): Implement this?
+int QuicSession::OnPathValidation(ngtcp2_conn* conn,
+                                  const ngtcp2_path* path,
+                                  ngtcp2_path_validation_result res,
+                                  void* user_data) {
+  auto session = static_cast<QuicSession*>(user_data);
```

Comment:
```suggestion
  QuicSession* session = static_cast<QuicSession*>(user_data);
```

Just to keep it consistent

Reply:
👍